packages/node_modules/@webex/internal-plugin-device/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@webex/internal-plugin-devices",
-  "version": "1.80.96",
+  "name": "@webex/internal-plugin-device",
+  "version": "1.80.95",
```

Comment:
I think there's a package conflict here that needs to be updated.

Reply:
I'm not sure what package `@webex/internal-plugin-device` collides with, please let me know if this is the case.

Reply:
Resolved with a rebase!